EDMarketConnector.py

```diff
@@ -41,74 +57,115 @@
 
     args = parser.parse_args()
 
+    journal_dir: str = config.get('journaldir') or config.default_journal_dir
+    # This must be at top level to guarantee the file handle doesn't go out
+    # of scope and get cleaned up, removing the lock with it.
+    journal_dir_lockfile = open(join(journal_dir, 'edmc-journal-lock.txt'), mode='w+', encoding='utf-8')
```

Comment:
Need a try/except here for the case of Journal files being shared read-only.  In that case we just assume we're the only process running and keep going.  That will be no different to running on !win32 in current `main`.